mmhuman3d/core/cameras/cameras.py

```diff
@@ -1190,10 +1252,13 @@ def compute_orbit_cameras(
     orbit_speed: Union[float, Tuple[float, float]] = 0,
     dist_speed: Optional[float] = 0,
     convention: str = 'pytorch3d',
-):
+) -> Union[torch.Tensor, torch.Tensor, torch.Tensor]:
```

Comment:
Union type; Union[X, Y] means either X or Y.

Reply:
Yes should be Tuple